### DISCUSSION TITLE
Flag unsaved page

### DISCUSSION THREAD

**Discussion by quazardous** [zim-desktop-wiki/zim-desktop-wiki#2002] [Ideas]
Unsaved pages could be flagged in the navigation tree and tabs.

**Comment by introt** [comment]
Pages should be saved automatically in the background, is this not happening on your system?

The following should show up in the debug log (Tools > View debug log) after a page is modified:

```
DEBUG: Store page in background: Foo:Bar
DEBUG: Index file: /home/introt/Notebooks/Test/Foo/Bar.txt
```

**Comment by quazardous** [comment]
I didn't catch that sorry !
